include/stl2/view/all.hpp

```diff
@@ -53,9 +52,8 @@ STL2_OPEN_NAMESPACE {
 	}
 
 	namespace ext {
-		template <Range Rng>
-		requires std::is_lvalue_reference_v<Rng> || View<decay_t<Rng>>
-		using all_view = std::decay_t<decltype(view::all(std::declval<Rng>()))>;
+		template <class Rng>
+		using all_view = decltype(view::all(std::declval<Rng>()));
```

Comment:
I prefer to leave the explicit `requires` clause. It will give better error messages.

Reply:
Yeah, I overreacted here when I noticed that `View<decay_t<Rng>>` should be `View<__f<Rng>>`. Will fix.